web/src/locales/fr/data.json

```diff
@@ -235,6 +237,8 @@
     "Email or phone": "Courriel ou téléphone",
     "Forgot password?": "Mot de passe oublié ?",
     "Logging out...": "Logging out...",
+    "Invalid Email or phone": "E-mail ou téléphone invalide",
+    "Login method": "Login method",
```

Comment:
The "Login method" widget has been removed. There should no this translation any more. Re-generate the i18n files

Reply:
Well I do not think it is a good idea, because it seems that the translation go-test doesn't work well and Engish text is directlly inserted into i18n files of all languages. Similar problems can also be found in recent PRs, So I think it may be not proper to regenerate i-18 files until this is fixed. Redundant items can be removed by i18 go tests, but it seems that false transation will not.

Reply:
So my suggestion is to remain as it is to avoid false translation from being translated, and fix it together after i18 translation is fixed